### PR TITLE
Update supported windows versions

### PR DESCRIPTION
**Pull request by chiyao-ms**
The old supported information is really confused for customer. Also, should we delete those un-supported windows version information?

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 1]
```diff
@@ -69,8 +69,11 @@ When non-destructive PIN reset is enabled on a client, a 256-bit AES key is gene
 Using Group Policy, Microsoft Intune or a compatible MDM solution, you can configure Windows devices to securely use the **Microsoft PIN Reset Service** which enables users to reset their forgotten PIN without requiring re-enrollment.
 
 >[!IMPORTANT]
-> The Microsoft PIN Reset service only works with **Enterprise Edition** for Windows 10, version 1709 to 1809 and later, and Windows 11. The feature works with **Enterprise Edition** and **Pro** edition with Windows 10, version 1903 and later, Windows 11.
-> The Microsoft PIN Reset service is not currently available in Azure Government.
+> The Microsoft PIN Reset service works with following versions of Windows.
+>
+> Windows 10 Enterprise Edition, version 1709 to 1809
+> Windows 10 Enterprise Edition and Pro edition, version 1903 and later
+> All Editions of Windows 11
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:76]:
```suggestion
> The Microsoft PIN Reset service works with the following versions of Windows:
>
> - Windows 10 Enterprise Edition, versions 1709 to 1809.
> - Windows 10 Enterprise Edition and Pro edition, version 1903 and later.
> - All Editions of Windows 11.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chiyao-ms, kindly commit the suggested changes. Many thanks.

**Comment by aczechowski** [comment]
Assigning to Paolo for final review. I'd suggest just removing the specific versions, which are all now end of service. So it could just be all supported versions of Windows 10 and Windows 11.

Also I noticed the line about Azure Government was removed - purposeful or accidental?

**Comment by paolomatarazzo** [comment]
Hello @chiyao-ms and thank you for the contribution. I updated the article as part of a different PR therefore I'm closing this one.

**Pull request closed without merging**